### ISSUE TITLE
Enhancement: DLL: interactive mode

### ISSUE THREAD

**Issue by reeelix** [cppcake/visu_alpro#18]
Proposal: Add a mode, where the `to_remove_pointer` can be moved to other nodes interactively. 
This can be done by two buttons: `move to prev`, `move to next`

Additionally, it would be very nice (but i see no obvious good way to implement it) if the order of pointer changes could be customized. 

Reason: in tutorials, i want to be able to visualize the operations a student tells

**Comment by neiroukh** [comment]
Yup, that's a great idea. I'd like to make a `playground`, in which you can add nodes and pointers and move any pointer any way you like. But who knows when ill do that...
